markdown/chapters/meandeck/core-concepts.md

```diff
@@ -0,0 +1,194 @@
+---
+authors: Doishy
+order: 0
+title: Core Concepts and Syntax Used
+---
+
+## Table of Contents
+
+## Preamble
+
+Welcome to Chapter 0 of the Doomsday section of the Wiki.
+
+This is the start of your journey into trying to learn a bit more about the deck
+and hopefully will inspire and help you to progress further with it!
+
+## Core Concepts
+
+### We're Teaching People to Fish
+
+As the Wiki front page states, "This is specifically not a primer as you often 
+see for other decks in Legacy, because primers generally become outdated rather
+quickly and are more about card choices, decklists, sideboard plans, and so on.
+The Doomsday Wiki is there to help people to really understand how the deck 
+works and can win in various scenarios.
+
+As a result a lot of the information presented within these chapters is to be
+taken as guidance only. The format (and the deck) changes as new cards are
+printed or existing ones banned/unbanned and, with the speed with which 
+conversation and innovation can occur, vastly outpaces the collective authors'
+ability to keep this site up to date. We try to keep things as high level / 
+universally applicable as possible. As a result do not seek to try and rote
+memorise any *piles* presented here but instead try and use them to ingrain core
+concepts or ideas into your play which you can then supplement with experience
+as you practice. There are plenty of people out there who will happily share 
+tables of *piles* with which you can use however it might be worth taking a few
+losses early on whilst learning the deck and trying to understand what you are
+doing rather than winning but getting railroaded in only a few scenarios. 
+
+### There are No Stupid Questions, and Very Few Original Ones.
+
+Questions are good. Quizzing things helps learning and understanding and if you
+don't start making inquiries into why things are as they are, then you're never
+going to get the most out of any deck you play. Conversely though, a lot of the
+questions you might want to ask are very likely to have been asked before. This
+is great as it means it is hopefully written down somewhere already! If
+you think of something you want answered and cannot find it in the chapters here
+there are a few places you can look which might help. First and foremost is the
+FAQs! (Frequently Asked Questions). These can be found in the appendices of the
+Wiki or by clicking [HERE!](markdown/appendices/faq). If it's not present there
```

Comment:
Path to FAQ page is incorrect and currently 404s. Should be _markdown/chapters/appendices/faq_